README.md

```diff
@@ -48,16 +68,16 @@ Make sure you replace `<bucket-name>` and `<your-ARN>` below with the appropriat
             "Resource": "arn:aws:s3:::<bucket-name>/public/*"
         },
         {
-            "Sid": "AllowWebsiteOnlyToProtected",
+            "Sid": "ServerFullAccess",
             "Effect": "Allow",
             "Principal": {"AWS":"<your-ARN>"},
-            "Action": "s3:GetObject",
-            "Resource": "arn:aws:s3:::<bucket-name>/protected/*"
+            "Action": "*",
```

Comment:
We shouldn't allow access to all actions, just the ones that we actually require to manage assets.